install.sh

```
@@ -263,6 +264,65 @@ setup_cursor() {
+
+  mkdir -p "$hooks_dir"
+  cp "$TMP_SCRIPTS/codex/hooks/session-start.sh" "$hooks_dir/"
+  cp "$TMP_SCRIPTS/codex/hooks/stop.sh" "$hooks_dir/"
+  cp "$TMP_SCRIPTS/codex/hooks/mnemo-protocol.md" "$hooks_dir/"
+  chmod +x "$hooks_dir/session-start.sh" "$hooks_dir/stop.sh"
+  ok "Hook scripts installed to ${hooks_dir}"
+
+  touch "$codex_config"
+
+  if grep -q '\[mcp_servers\.mnemo\]' "$codex_config" 2>/dev/null; then
+    ok "~/.codex/config.toml: mnemo MCP already configured"
```

Shellcheck (0.11.0)

[warning] Tilde does not expand in quotes. Use $HOME.

(SC2088)